projects/dev/src/app/group/group-manage/pending-request/pending-request.component.ts

```diff
@@ -76,11 +149,73 @@ export class PendingRequestComponent implements OnInit, OnChanges {
   onExpandWidth(e) {}
 
   onClickAccept(e) {
-    this.onAccept.emit(e);
+    if (
+      this.selection.length === 0 ||
+      this.acceptLoading ||
+      this.rejectLoading
```

Comment:
We should do this. 
Because the `click` event already bind to the `accept/reject` button and we should prevent clicking it again logically with this guard.

Reply:
Ok, that answers my question. I'll recheck with the new version of the code...